updater/lib/dependabot/setup.rb

```diff
@@ -37,7 +37,31 @@
     pub
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = lambda do |event, hint|
+    debugger
+    if !hint[:exception].nil?
+      ExceptionSanitizer.process_sentry(event, hint)
+
+      # TODO integrate these two...
+      # TODO copied from migration guide, I don't understand how this assignment works though?
+      # https://docs.sentry.io/platforms/ruby/migration/#exceptionraven_context
+      if exception = hint[:exception]
```

Comment:
Ruby n00b question: What does this construct do?

I see inline assignment, does it also check if `hint[:exception]` is nil or does it check if it's false?

Reply:
It checks whether `hint[:exception]` is either `nil` or `false`, that's usually called "falsy".